### ISSUE TITLE
Running some tasks only once

### ISSUE THREAD

**Issue by capooti** [cga-harvard/HHypermap#286]
Some periodic task should run only once at a time. For example tindex_cached_layers

Here there is some good hint to get this done:
https://stackoverflow.com/questions/4095940/running-unique-tasks-with-celery

**Comment by JorgeMartinezG** [comment]
@capooti  Is the purpose to index cached layers one by one? I mean, the goal is to avoid search engine saturation, right?

If so, what about bulk indexing?

**Comment by capooti** [comment]
Yes, I am bulk indexing. In production that task is scheduled every 2 minutes. When checking services with many layers cache will be full of layers to index and it can happen that in 2 minutes that task is not completed, and there will be more ones running at the same time which can cause problems to solr
